src/WireMock/Client/WireMock.php

```diff
@@ -114,7 +125,7 @@ public function verify($requestPatternBuilderOrCount, ?RequestPatternBuilder $re
             $numberOfRequestsMatcher = null;
         }
 
-        $requestPattern = $patternBuilder->build();
+        $requestPattern = $patternBuilder->build(); // @phpstan-ignore method.nonObject (@TODO this looks like an actual bug to me?)
```

Comment:
so this looks like a bug to me.

if its an int then $pattnerBuilder is $requestPatternBuilder;

but this can be null and is not checked here? before its called.